src/hAPI/README.md

```diff
@@ -1,5 +1,54 @@
-# hAPI TypeScript Library 
+# What is this?
 
-## Acknowledgements 
+This repository contains the necessary components to run the multimodal audio-haptic photo renderer in the IMAGE extension using the Haply 2DIY device.
 
-The [hAPI library](https://github.com/Shared-Reality-Lab/IMAGE-browser/tree/main/src/hAPI) used in the project is a TypesSript port of the [Java hAPI library](https://gitlab.com/Haply/hAPI) originally authored by Oliver Anthony, [Haply Robotics](https://haply.co/). 
+## Requirements
+
+* A Haply 2DIY with its firmware installed.
+## How To Use
+
+* Ensure the Docker container hosting the appropriate haptic handler is running on the development / main server.
```

Comment:
What's the appropriate handler? Can you give an example?

Reply:
I intended this to initially be generic, but then realized the web worker code is significantly tied to our photo-audio haptics handler implementation, so I linked to that (which has its own documentation as well).